pages/docs/essentials/importing.mdx

```diff
@@ -11,10 +12,30 @@ in most cases, installing the `effect` package is all you need to get started. L
 
 To begin your journey with Effect, you first need to install the `effect` package. Open your terminal and run the following command:
 
+<CustomTabs items={["NPM", "PNPM", "Yarn"]}>
+<Tab>
+
 ```bash filename="Terminal"
 npm install effect
 ```
 
+</Tab>
+<Tab>
+
+```bash filename="Terminal"
+pnpm install effect
```

Comment:
shouldn't here be `pnpm add effect`?

from `pnpm -h`:

|            |                                                                                                                       |
| ---------- | --------------------------------------------------------------------------------------------------------------------- |
| add        | Installs a package and any packages that it depends on. By default, any new package is installed as a prod dependency |
| i, install | Install all dependencies for a project                                                                                |

Reply:
Fixed 👍 

Sorry, I read that pnpm was a drop-in replacement for npm & assumed the equality of subcommand names.